dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
+        || DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+    {
+        if (auto backButton = m_backButton.try_as<winrt::Button>())
+        {
+            backButton.Width(40);
+        }
+        if (auto paneToggleButton = m_paneToggleButton.try_as<winrt::Button>())
+        {
+            paneToggleButton.MinWidth(40);
+        }
+        if(auto paneButton = m_paneToggleButton.try_as<winrt::Button>())
+        {
+            if (auto paneToggleButtonIconColumn = paneButton.GetTemplateChild(c_paneToggleButtonIconGridColumnName).try_as<winrt::ColumnDefinition>())
+            {
+                auto width = paneToggleButtonIconColumn.Width();
+                width.Value = 40;
+                paneToggleButtonIconColumn.Width(width);
```

Comment:
Why is this needed? why not just paneToggleButtonIconColumn.Width(40)

Reply:
Because Width is of type GridLength. So we either need to create a new GridLength or modify the old one, and that's what we are doing here.

Reply:
Can you try panToggleButtonIconColumn.Width({40}) or maybe panToggleButtonIconColumn.Width(winrt::ColumnWidth{40});

Reply:
Unfortunately neither of those did work as the current code does.

Reply:
any idea why? we should be able to construct a new GridLength via winrt instantiation.

Reply:
I've tried different ways to get this to work but I was not able to just use winrt instantiation here for some reason :(